src/core/ui/components/drop-down/googleIcon.tsx

```diff
@@ -0,0 +1,31 @@
X1DA1NPlE+s7E/LSEwKLQ3CUCGKIpjkzaYQMsAazc40wEpECmVck/YQbaKEqaXJm+ZMmrlj7tTZ2ye3d+2/qDT/wRUzlrAJO405DLCIiBrHEZSIqEkYYNFE99TBLa0v9e1fsXvwwMrdAwdW7hvsXnq80ruwZjNIv7bcr8MAazQ7wwBLnxPM3jq1fcr+c6fM2TJ/8tzNZ7fM2H75Wcu7J8jmo3GMARYRUeM4ghIRNQkDLKJXe+rg8637eg4tfWXgxPyN6XN3lMuDXT2DJ1aUkzIykUIo7au0NDSkjPLXkLaAlZBQ/p/wLyuJTBkIayCM9W923PXSd9zK3/rU3Pe7pltuJTuhIeFuZ2Fco3ljEJiSP9G0/mKgJWCs9d/jorNY5w3FXZDmejJpWP94rLD5qnnW3dsp9jd8x3Xy5Lb+uE8nX3fIkK8aSwTyflEmyPxzPn2YkTYP9fzjlPZUT6n643M/29R/qE0rUCr0PaiEUDA6v73bzvljPHVf9QfstsTJnxUKCW3z73HXu/s3p22HpGQRIUCrLqDDFDEtmPTw2W0zt54zdc7WGR2Td9963vWb+PKg8YgBFhFR4ziCEhE1CQMsol/viQOPd3UP9cw7NHR88cGhY4v3l48tPTx49Kb+yhCqonwygApcYFQPZFwA5Vatc72SfKjkAh6ZBzPafay/9lpt6D+6VRC1zUOokZUQXTDlQ7KR4EtbKAsEEP46BYE+kQc5oh5wvSpEEsI3qsfrvM51/f/xaYnW653QVqw+dV/1r59+uwyvbmw+8jUf0bnHpbOT1+eP8VS45b5HmbzHlHt+7v5dsCd9yCf815OokG8r6Hqz/ZGArv5YTNXfbyBkHhrWH4e7D3f9YFjzYZfb8KEWCBD5fVIKiyiE8fYLcNamrvZJh2Z1zdxx1qQZO6cXp+y9ZPpiTgmkcY8BFhFR4ziCEhE1CQMsot/OE93bOk4MD84dSiod2048t7ZnqH/WKwPH5h8dOjG3Vw/Nq8gESWRglEWbEXnwUp9pKEVQP3HMA5pCkuYVVMKtcudqswSsyYMuV3GUoeq/z1pxcrLiSBDkRPWAyl0nLV4TYEn/c4U99X0jFUv+cytRkdlptz/1dmzkfuJ6KZQL3177NfcxkHlodrJiytpXzZqMbHjyeh/OWY3M5t/jv68Qv6p6y1Wqnf4zVFK/M2n9dlKB8J+PfL8WSb5NlfTZVaotMm39dnRKVqA1KGJq3LF1Rjxp95zC1B1z26Zvnztp5vau1s7uS2Zd1s/DnyYiBlhERI3jCEpE1CQMsIjeGj8/tGXW4aETC/cPdC892Nu9+ES5d+5h/cr8JEvnJ0kNqdGnTfXLK4p0ZPLwydclCR9UuWl7fmVEYxFZVQ+sFIwaqeAaqaCSCG315GMX9QBL2FMhlWtC/qq3WVbmlVT120CcqpB6rXzanTz5tTyAOhUwuUs0EnCJU0HaSLjmvy87/TrjK8tQD8Tc9aWq8c/JbxMXSEn4ajV3O1dlJQOd/8wsr65yz9GHXPXHUQ5jRDZAbBVaEWFS0LJlWkvX/untk3e3t7QeWxKdvbGtpfXYlI4p+5dOW6x5qBPlGGARETWOIygRUZMwwCJ6+/zboS1zB6tDXceH+uYeHT4x72jlxLzjld65PZXBW4eSYfSiijRNkWQ1ZFZDBAZ+hl29yqgwEjTVe1H5qXFW+hDHfT1V8aum74301ho5ORXqVMB1+kvd1oMnhV+f6ejXmTb4Kln0q9MHhTn18+Vp/a5sPZhDvW+XD7OqI8nYydu4HmLu6+4+qiJFCIVQBIhEjKKM0Rq2oK3QikJUvG9WOGVHZ6Gte2rbpP3TWibtnVpo37ti5iWcAkj0JhhgERE1jiMoEVGTMMAiemc9dfzZ1nJa7qpmtdaecnlWuTbU1Ts0MKOn2jerLx2c0Z+WZwxUy2uH0wr6RL/vpZVp7duTm3qT8zz8MQgRngqHfKOsvHeWqVdEqcqpAGqkQTpOO3lN5akA67X9s5xA16u5ThsmxGm3y0rm5HU+FKs/FqHr9496hVc9NHPBm/T9tKSf4jgoIx9QFWSAkgjQImOUggglVdgYq6B8VmnqzmJYLLcXO451FNq7JxU6utvjtmPtxbZjhahYvqjj7ISHL9FvjwEWEVHjOIISETUJAyyi0eWZo89Gg0l1ai2tFrtt7/xqUmntqwzN6K0MzBqolaf2VodmDNaGu2pZ2jqU9HQYY+YmOkFiXcilkdrMN4Q3RiMOOv1z8/3N3Vf9yn/mZBP3gqivFmhONXsfabjuK6BOD7Vc+IS8mfvIyW8F/SerwpRvLl9fibE+TTGOIkQqQBwW9hZUXC4EYbmgonIcFiqhCirnF8/dHAdhuTUq9bRFxZ62uOVYW1zsaQtLxy6ediHDKaK3CQMsIqLGcQQlImoSBlhEY9ezx3apLMuiSlptHU4rHa6qq6prre5jZnR0NH1lnjFGpSYrpjqJUp0UE51/rmGj1NjIWqvcbU6fCiiE0H7lQRX6EElCaCWkDqWqhEGQuI/u/+2y65gSMomDqBIFcaUQFsqFoFCOg6gsZaDbZKEnVFGlFMb9S6fPZw8qolGCARYRERERERERERERERERERERERERERERERERERERERERERERERERERERERER/f/twYEAAAAAw6D5U1/gCBUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABzVAECCLyGXbTCWAAAAAElFTkSuQmCC"
```

Comment:
Is this loadable from a remote resource? This seems really heavyweight here, no?

Reply:
yeah thats a looong href haha
updated